springerlink.md

```diff
@@ -3,8 +3,8 @@
 ### **VPAT**&reg; **Version 2.2 – July 2018**
 
 **Name of Product/Version**: SpringerLink _(we do not version our software)_  
-**Product Description**: SpringerLink provides researchers with access to scientific documents from journals, books, series, protocols, reference works and proceedings. This report refers entirely to the accessibility of all content published at the subdomain link.springer.com, including informational pages, and all content found at checkout.springer.com.  
-**Date**: 27 February 2019  
+**Product Description**: SpringerLink provides researchers with access to scientific documents from journals, books, series, protocols, reference works and proceedings. This report refers entirely to the accessibility of all web content published at the subdomain link.springer.com, including informational pages, and all web content found at checkout.springer.com and order.springer.com.  
```

Comment:
> SpringerLink provides researchers with access to scientific documents from journals, books, series, protocols, reference works and proceedings

I'm wondering if this bit should be updated to include video, conference series and protocols. Although this was taken from the meta description so maybe that part needs updating. 🤔

> This report refers entirely to the accessibility of all content published at the subdomain link.springer.com

Should rd.springer.com be mentioned? Everything is the same as link.springer.com aside from the logo at the top of the page and an extra set of links in the 'Industry' tab on the home page. You have to use the edition switcher (will say either display academic edition or corporate edition) to view 'rd' or 'link' version of the site otherwise will redirect you back what was originally set by cookie.

Reply:
Yeah, both of these make sense. I guess the meta description _and_ the VPAT both need updating to include videos. "Protocols" and "series" are already in there - or are there separate "conference series and protocols" in addition to the ones that are already covered? 

I do think that rd.springer.com should be mentioned. Will update in a separate commit, thanks! 👍

Reply:
Oh you're right, protocols and series already there 😬

Reply:
Just adding a note that we decided to return to adding videos to the description at the next review, as the german language version will also need to be taken into account